### ISSUE TITLE
Shaman - Earth shield and Glyph of Totem of Wrath do not fade with respec.

### ISSUE THREAD

**Issue by kysqt** [dalaranwow/dalaran-wow#5213]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Spells like Earth shield and Glyph of Totem of Wrath do not fade when the shaman changes spec

**Expected behaviour**: Spells effected by talents shouldn't remain active after a shaman gets rid of said talents.

**Steps to reproduce the problem**: 

1. Make shaman
2. Get duel spec
3. 

**Include proofs for this behaviour**
https://i.imgur.com/yzydirh.jpg
https://i.imgur.com/9nIujCE.jpg
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Earth Shields:
https://wowgaming.altervista.org/aowow/?spell=974
https://wowgaming.altervista.org/aowow/?spell=32593
https://wowgaming.altervista.org/aowow/?spell=32594
https://wowgaming.altervista.org/aowow/?spell=49283
https://wowgaming.altervista.org/aowow/?spell=49284

Glyph of Totem of Wrath:
https://wowgaming.altervista.org/aowow/?spell=63926

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
https://i.imgur.com/yzydirh.jpg
https://i.imgur.com/9nIujCE.jpg

You can clearly see I'm resto with Glyph of Totem of Wrath still active and Elemental with Earth shield on my focus. I'm sure this effects more classes than shamans.

**Comment by TeridanDW** [comment]
Hello and thank you for the report @kysqt.
Issue confirmed on PTR.
This:

> I'm sure this effects more classes than shamans.

is true. All talents that adds/casts any spells on player/target/friendly should be removed on respec/spec switch.
Diffrent example for this issue is spell https://wowgaming.altervista.org/aowow/?spell=49016 not been removed from a friendly target when the DK switches specs.

**Comment by TeridanDW** [comment]
After a quick test on PTR:
 - Earth shield is been removed on spec change only if it was casted on self, if not it stays on the friendly target (incorrect - should be removed even on friendly targets)
 - Spell https://wowgaming.altervista.org/aowow/?spell=63283 is not been removeds on spec change at all.
